### ISSUE TITLE
[enhancement] The keyboard navigation should be more visible

### ISSUE THREAD

**Issue by obotisan** [mozilla/blurts-server#1212]
[Affected versions]:
- Firefox 69.0.1

[Affected Platforms]:
- Windows 10 x64
- macOS 10.13
- Ubuntu 18.04 x64

[Steps to reproduce]:
1. Navigate to https://monitor-v2.herokuapp.com/
2. And try to open the Bento menu using the keyboards.

[Expected result]:
- You can follow easily where the focus is on while pressing on the Tab key. 

[Actual result]:
- It's really hard to see where the focus is on the page.  

[Regression]:
- N/A

[Additional Info]:
- A higher contrast between the colors might help. Or maybe this happens because you can almost never tell on which element the focus is on. 
- The only way to see if the focus is on bento box button is if you keep your eyes on it while navigating slowly with the keyboard.

**Comment by obotisan** [comment]
Another thing is that when the focus is on "Made by Mozilla" you can't see it at all.

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
